compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -516,10 +516,16 @@ pub static BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     // ==========================================================================
 
     // Linking:
+    #[cfg(bootstrap)]
     gated!(
         naked, Normal, template!(Word), WarnFollowing, EncodeCrossCrate::No,
         naked_functions, experimental!(naked)
     ),
+    #[cfg(not(bootstrap))]
+    gated!(
+        unsafe naked, Normal, template!(Word), WarnFollowing, EncodeCrossCrate::No,
+        naked_functions, experimental!(naked)
+    ),
```

Comment:
Is this still needed since the bootstrap issue with builtins was resolved?

Reply:
It should not be, but let's see what CI thinks.